pkg/workspace/workspace.go

```diff
@@ -68,11 +74,25 @@ func openWorkspace(fsys fs.FS, rootPath string) *Workspace {
 //
 //    - warpforge-error-workspace -- when the workspace directory fails to open
 func OpenHomeWorkspace(fsys fs.FS) (*Workspace, error) {
-	return OpenWorkspace(fsys, homedir)
+	workspace, err := OpenWorkspace(fsys, homedir)
+	if err == nil {
+		workspace.isHomeWorkspace = true
+		workspace.isRootWorkspace = true
+	}
+	return workspace, err
+}
+
+// DotPath returns the workspace's path including the .warp* segment.
+func (ws *Workspace) DotPath() string {
```

Comment:
I'm not a huge fan of `DotPath` as a name because it describes the path format rather than what it does. Something like `InternalsPath` would be more descriptive of function.

Reply:
Good point, I'll rename it to this.